File: java/MASTER_SLAVE_WHOLE_PART_PROXY/Cargo.java
public class Cargo{
    //Atributos
    private int cargo;
    private boolean reembolsable;
    private boolean cargoExitoso;

    //Constructor
    public Cargo(){}

    //Métodos
    public void setCargo(int cargo, boolean reembolsable){
        this.cargo = cargo;
        this.reembolsable = reembolsable;
        this.cargoExitoso = false;
    }

    public void setCargoExitoso(boolean cargoE){
        this.cargoExitoso = cargoE;
    }

    public boolean getcargoExitoso(){
        return cargoExitoso;
    }

    public int getCargo(){ //no se usa
        return cargo;
    }

    public boolean getReembolsable(){ //no se usa
        return reembolsable;
    }
}

File: php/MASTER_SLAVE_WHOLE_PART_PROXY/pago/Cargo.php
<?php
class Cargo {
    private $cargo;
    private $reembolsable;
    private $cargoExitoso;
    
    public function __construct() {
        
    }
    
    public function setCargoSolicitado($cargo, $reembolsable){
        $this->cargo = $cargo;
        $this->reembolsable = $reembolsable;
        $this->cargoExitoso = true;
    }
    public function getCargo() {
        return $this->cargo;
    }

    public function getReembolsable() {
        return $this->reembolsable;
    }

    public function getCargoExitoso() {
        return $this->cargoExitoso;
    }

    public function setCargo($cargo): void {
        $this->cargo = $cargo;
    }

    public function setReembolsable($reembolsable): void {
        $this->reembolsable = $reembolsable;
    }

    public function setCargoExitoso($cargoExitoso): void {
        $this->cargoExitoso = $cargoExitoso;
    }




}
